templates/create_meeting.html

```diff
@@ -35,42 +34,37 @@
 .container {
   border-radius: 5px;
   background-color: #f2f2f2;
-  padding: 20px;
+  padding: 15px;
 }
-
-.ui.fluid.dropdown{display:block;width:100%;min-width:0}
-
 </style>
-</head>
-<body>
 
-<h3 align="center">Create a Meeting for "ProjectNameHere" </h3>
+<br><h3 align="center">Meeting Creation for {% if project %}{{project.project_name }}{% endif %} </h3>
 
 <!-- Meeting creation form -->
 <div class="container">
   <form autocomplete="off" action="/action_page.php">
```

Comment:
This is broken (action_page.php), but I'm guessing that's a future feature. (Assuming that the value will be changed, since this isn't a PHP project ^_^)